features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml

```diff
@@ -2131,6 +2131,13 @@
     <!--This is the separator that use to separate multiple roles in the role claim value coming from IDP side-->
     <FederatedIDPRoleClaimValueAttributeSeparator>,</FederatedIDPRoleClaimValueAttributeSeparator>
 
+    <!--This enables federated user's role based authorization -->
+    <EnableFederatedIDPRoleBasedAuthorization>false</EnableFederatedIDPRoleBasedAuthorization>
```

Comment:
Merge these two configs & if at least one application exist consider as feature is enabled